showcases/shared/custom-select.json

```
@@ -446,6 +446,59 @@
 			}
 		]
 	},
+	{
+		"name": "Show Required Asterisk",
```

```suggestion
		"name": "Show asterisk for required form fields (custom select)",
```

I'm struggling whether it might be easier to understand when explaining it with some more words.

---

https://github.com/db-ux-design-system/core-web/pull/4509#discussion_r2196900438